BuildScript.Nuke/MSBuildProperties.cs

```diff
@@ -36,4 +36,5 @@ public static class MSBuildProperties
   public const string AssemblyInfoFile = "AssemblyInfoFile";
   public const string RestorePackagesConfig = "RestorePackagesConfig";
   public const string SolutionDir = "SolutionDir";
+  public const string ContinuousIntegrationBuild = "ContinuousIntegrationBuild";
```

Comment:
should we switch those to MyConst = nameof(MyConst)?

Reply:
I don't think that is a good idea since it makes it easier to break the loose binding with the MSBuild part by renaming the property.